File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int ans=0;
        for(int i=0;i<nums.size();i++)
        {
            int count=0;
            while(i<nums.size() && nums[i]==1)
            {
                count++;
                i++;
            }
            if(count!=0) i--;
            ans=max(ans,count);
        }
        return ans;
    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxi=0;
        int k=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]==1) k++;
            else k=0;
            maxi = Math.max(maxi,k);
        }
            return maxi;
    }
}